src/services/catalog.service.ts

```diff
@@ -55,7 +55,21 @@ export class CatalogService {
 
     // Perform a case-insensitive sort on display name, falling back to kind
     // and metadata.name for a stable sort when items have the same display name.
-    return _.sortByAll(items, [(item: any) => item.name.toLowerCase(), 'resource.kind', 'resource.metadata.name']);
+    items = items.sort((item1: any, item2: any) => {
+      let comparison = item1.name.localeCompare(item2.name, undefined, {sensitivity: 'base'});
+
+      if (comparison === 0) {
+        comparison = item1.resource.kind.localeCompare(item2.resource.kind, undefined, {sensitivity: 'base'});
```

Comment:
I'd feel more comfortable using `_.get` here and below to avoid an error if the property doesn't exist.

```js
_.get(item 'resource.kind', '')
```

https://lodash.com/docs/3.10.1#get